src/screens/Interests.tsx

```diff
@@ -42,9 +48,43 @@ function Interests() {
 
   };
 
+  useEffect(() => {
+    // fetches resource-lists to use the category
+    const fetchResourceData = async () => {
+      try {
+        const res = await fetch(APIUrl + "resource-lists");
+        const json = await res.json();
+        let jsonData = json.data.map((x: any) => {
+          return {
+            category: x.attributes.category,
+            sub_category: x.attributes.sub_category,
+            link: x.attributes.link,
+          };
+        });
+
+        // only extract unique categories
+        const categoryTitles = Array.from(
+          new Set(jsonData.map((x: any) => x.category))
```

Comment:
If you make jsonData typed you won't need to specify it as an `any` which will be a lot safer